### DISCUSSION TITLE
How to listen to nested PropertyChanged events of Observable collection in Parent class

### DISCUSSION THREAD

**Discussion by prabhavmehra** [CommunityToolkit/dotnet#838] [Q&A]
Suppose I have a class A which extends `ObservableObject` which has an ObservableCollection. The ObservableCollection is of type Model `ObservableCollection<Model>`.

This model extends `ObservableObject` and has some `ObservableProperty`. When I edit any of the properties of an item I can see the change UI from the observable collection. 

Is there a way I can hook into the Property changes of any instance of Model class within the ObservableCollection in Class A?

So basically
```c#
// Class A

PropertyChanged += OnPropertyChanged;

// This is triggered when I edit an element of type model from the ObservableCollection
void OnPropertyChanged(...){
   // Do something
}
```
